pkg/operator/operator.go

```diff
@@ -10,24 +10,24 @@ import (
 	"math/rand"
 	"net/http"
 	"net/url"
-	"os"
 	"sync"
 	"time"
 
-	"github.com/davecgh/go-spew/spew"
-	"github.com/prestodb/presto-go-client/presto"
-	promapi "github.com/prometheus/client_golang/api"
-	prom "github.com/prometheus/client_golang/api/prometheus/v1"
-	"github.com/prometheus/client_golang/prometheus/promhttp"
-	log "github.com/sirupsen/logrus"
-	"github.com/taozle/go-hive-driver"
+	"github.com/kube-reporting/metering-operator/pkg/db"
+	cbClientset "github.com/kube-reporting/metering-operator/pkg/generated/clientset/versioned"
```

Comment:
If we're tackling this kind of work now, I would expect to also rename this import and all of its occurrences too, to something like `meteringClientset` or anything that doesn't involve the old "chargeback" name.

Reply:
> If we're tackling this kind of work now

We don't have to merge this now, I think it should be no-op so it shouldn't be too risky but bugs first and we can merge these when master opens if that't the right thing to do.



> I would expect to also rename this import and all of its occurrences too, to something like `meteringClientset` or anything that doesn't involve the old "chargeback" name.

Easy enough, I'll add a new commit for this refactor